magda-typescript-common/src/registry/TenantConsts.ts

```diff
@@ -0,0 +1,5 @@
+// TODO: Investigate how to use swagger codegen to automatically generate these constants.
+// These constants are the same as defined in the scala model Registry.
+export const MAGDA_TENANT_ID_HEADER = "X-Magda-TenantId";
```

Comment:
Could this be `X-Magda-Tenant-Id` so we're not mixing CamelCase with kebab-case?

Reply:
Will fix it.

Reply:
done